package.json

```diff
@@ -59,14 +58,22 @@
     "ldapjs": "1.0.1"
   },
   "devDependencies": {
+    "babel-core": "6.21.0",
+    "babel-loader": "6.2.10",
+    "babel-preset-es2015": "6.18.0",
     "chai": "3.5.0",
     "eslint": "3.12.2",
     "font-awesome": "4.7.0",
     "handlebars": "4.0.6",
```

Comment:
So far `handlebars` in `devDependencies` was used just for testing, but now it is used to build the bundle. Version we had in the repo was v4.0.5. Are we ok going with v4.0.6 as part of this PR or play it safe and go one change at a time?

FYI diff is at https://github.com/wycats/handlebars.js/compare/v4.0.5...v4.0.6.